src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/CaseRolesDroolsTest.java

```diff
@@ -207,14 +235,23 @@ private void verifyGrantOrRejectAccessFor_CaseRole(String jurisdiction, String c
     @ParameterizedTest
     @CsvSource({
         "SSCS,Benefit,hearing-judge",
+        "SSCS,Benefit,tribunal-member-1",
+        "SSCS,Benefit,tribunal-member-2",
+        "SSCS,Benefit,tribunal-member-3",
+        "SSCS,Benefit,appraiser-1",
+        "SSCS,Benefit,appraiser-2",
         "SSCS,Benefit,panel-doctor",
         "SSCS,Benefit,panel-disability",
         "SSCS,Benefit,panel-financial",
         "SSCS,Benefit,panel-appraisal-judge",
         "SSCS,Benefit,panel-appraisal-medical",
```

Comment:
Are these still required:
        "SSCS,Benefit,panel-doctor",
        "SSCS,Benefit,panel-disability",
        "SSCS,Benefit,panel-financial",
        "SSCS,Benefit,panel-appraisal-judge",
        "SSCS,Benefit,panel-appraisal-medical",

Reply:
Resolved in 83bf196b8348c5f9c90a2b71701377756c7b5f80.